CityOfDetroit/bloom
# UnitsSummaries in import script

## Issue

- Closes #410 

## Description

Adds the `unitsSummary` field to `listings` in our csv import script. The only calculation it performs is `totalCount` per `unitType`.

## Type of change


## How Can This Be Tested/Reviewed?

Start the backend and run the import script with HRD data. After running the script, I verified that the `units_summary` table in our database had the expected changes.

## Checklist:
